cinema/serializers.py

```diff
@@ -112,14 +112,14 @@ class Meta:
 
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
In the `validate` method of `TicketSerializer`, you are creating a Ticket instance and calling the `clean` method on it. This is a bit confusing because it seems like you are creating a new Ticket instance just to validate the data, but this instance is not saved or used anywhere else. It would be more efficient to validate the data directly without creating a Ticket instance.